### ISSUE TITLE
Update the API documentation

### ISSUE THREAD

**Issue by Pomax** [devongovett/fontkit#117]
The README.md contains some API examples (such as `font.getGlyphWidth`) that no longer apply/exist, so it might be worth taking the API docs out of the README.md entirely and instead generating it in a docs dir using a small docs generator based on documented code.

**Comment by devongovett** [comment]
Agree. I've used [esdoc](https://esdoc.org) on some other projects, and I like it.
